target/linux/bmips/bcm6368/base-files/etc/board.d/01_leds

```diff
@@ -6,6 +6,9 @@
 board_config_update
 
 case "$(board_name)" in
+actiontec,r1000h)
+	ucidef_set_led_usbport "usb" "USB" "green:usb" "usb1-port1" "usb2-port1"
+	ucidef_set_led_netdev "wan" "WAN" "green:wan" "wan"
```

Comment:
I've just noticed that this is missing `;;`, but don't worry, I'll fix it myself :)